libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -77,7 +79,15 @@ public ConfigService(SSMProvider ssmProvider, SecretsProvider secretsProvider) {
         this(ssmProvider, secretsProvider, null);
     }
 
-    public ConfigService(List<String> featureSet) {
+    @ExcludeFromGeneratedCoverageReport
+    public ConfigService() {
+        if (Boolean.parseBoolean(getEnvironmentVariable(UNMOCKABLE_CONFIG_SERVICE_TEST))) {
+            // For journey map tests that deserialize events that create a config service
```

Comment:
Is this because the `BasicState` initialiser constructs a new config service each time?

This feels a bit suspect - I wonder if we can put the `configService` into `JourneyContext` instead, to avoid the states themselves being stateful (ha). If that makes this PR too complicated then we can leave it like this...

Reply:
...interesting. That'll be doable, and remove the need for the tests flag...